Gemfile.lock

```diff
@@ -220,13 +220,8 @@ GEM
     multi_xml (0.7.1)
       bigdecimal (~> 3.1)
     netrc (0.11.0)
-    nokogiri (1.16.0-aarch64-linux)
-      racc (~> 1.4)
-    nokogiri (1.16.0-arm64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.16.0-x86_64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.16.0-x86_64-linux)
+    nokogiri (1.17.2)
+      mini_portile2 (~> 2.8.2)
```

Comment:
There's already 1.18.0.rc1 which already supports platform specific versions in Ruby 3.4. I suspect final version will be released very soon.

Reply:
Okay, let me upgrade to `1.18.0.rc1` for now, then we can bump to `1.18.0` once it's released

Reply:
Cool, it should work the same, just take longer to install.